File: app/src/main/java/com/xia/fly/demo/mvp/view/HomeView.java
package com.xia.fly.demo.mvp.view;

import com.xia.fly.demo.mvp.presenter.HomePresenter;
import com.xia.fly.mvp.BaseMvpView;

/**
 * @author xia
 * @date 2018/7/20.
 */
public interface HomeView extends BaseMvpView<HomePresenter> {

    void setData(String content);

    boolean getBoolean();

    int getNum();
}

File: app/src/main/java/com/xia/fly/demo/mvp/view/HomeView.kt
package com.xia.fly.demo.mvp.view

import com.xia.fly.demo.mvp.presenter.HomePresenter
import com.xia.fly.mvp.BaseMvpView

/**
 * @author xia
 * @date 2018/7/20.
 */
interface HomeView : BaseMvpView<HomePresenter> {

    val boolean: Boolean

    val num: Int

    fun setData(content: String)
}
